wiremock/__files/assets/stylesheets/footer.css

```diff
@@ -0,0 +1,181 @@
+.govuk-width-container {
```

Comment:
Copy and paste verbatim from the css that is served by the frontend components API